distsql/select_result.go

```diff
@@ -159,11 +159,13 @@ func (r *selectResult) getSelectResp() error {
 		if err := r.selectResp.Error; err != nil {
 			return terror.ClassTiKV.New(terror.ErrCode(err.Code), err.Msg)
 		}
+		sc := r.ctx.GetSessionVars().StmtCtx
 		for _, warning := range r.selectResp.Warnings {
-			r.ctx.GetSessionVars().StmtCtx.AppendWarning(terror.ClassTiKV.New(terror.ErrCode(warning.Code), warning.Msg))
+			sc.AppendWarning(terror.ClassTiKV.New(terror.ErrCode(warning.Code), warning.Msg))
 		}
 		r.feedback.Update(re.result.GetStartKey(), r.selectResp.OutputCounts)
 		r.partialCount++
+		sc.MergeExecDetails(re.result.GetExecDetails())
```

Comment:
It's better to print execution details for every coprocessor request.

Reply:
Sometimes there are hundreds of coprocessor requests, then the SLOW_QUERY log would be unreadable.

Reply:
sometime there is a data skew in some key ranges, and this data skew can slow down the coprocessor performance and cause other coprocessor request waits for processing. 

if we merge all the coprocessor request for a user query, we can not recognize the data skew.

Reply:
How about we maintain a max?

Reply:
The main problem this PR solves is to distinguish culprit queries among SLOW queries.
The total time is enough for this purpose.